server/schedulers/balance_region.go

```diff
@@ -130,6 +144,7 @@ func (s *balanceRegionScheduler) IsScheduleAllowed(cluster opt.Cluster) bool {
 func (s *balanceRegionScheduler) Schedule(cluster opt.Cluster) []*operator.Operator {
 	schedulerCounter.WithLabelValues(s.GetName(), "schedule").Inc()
 	stores := cluster.GetStores()
+	s.filters[1] = s.hitsCounter.buildSourceFilter(s.GetName(), cluster)
```

Comment:
kind strange to reset filters[1] every time. I prefer `filters := append(s.filters, s.hitsCounter.buildSourceFilter(s.GetName(), cluster))`

Reply:
done